recipes/r-multimode/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = '1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-multimode
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/multimode_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/multimode/multimode_{{ version }}.tar.gz
+  sha256: 9dea73b28162377dc7b4e3a0b99f0b36f850c5e48472963e562a791397145390
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-diptest
+    - r-ks
+    - r-rootsolve
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-diptest
+    - r-ks
+    - r-rootsolve
+
+test:
+  commands:
+    - $R -e "library('multimode')"           # [not win]
+    - "\"%R%\" -e \"library('multimode')\""  # [win]
+
+about:
+  home: https://arxiv.org/abs/1803.00472/
+  license: GPL-3
+  summary: Different examples and methods for testing (including different proposals described
+    in Ameijeiras-Alonso et al., 2018 <DOI:10.1007/s11749-018-0611-5>) and exploring
+    (including the mode tree, mode forest and SiZer) the number of modes using nonparametric
+    techniques.
+  license_family: GPL3
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - khughitt
```

Comment:
Please do not remove the commented cran metadata that is normally added here at the end. It helps reviewing the recipe.

Reply:
Okay. I was unsure about this as well, but based on #8 in https://conda-forge.org/docs/maintainer/adding_pkgs.html#step-by-step-instructions ("Remove all irrelevant comments in the meta.yaml file.") decided to take them out since they did not seem essential. I will regenerate the recipe and leave the comments in.

Reply:
Using https://github.com/bgruening/conda_r_skeleton_helper helps to remove the non-relevant comments and keep the relevant one.